include/exadg/structure/spatial_discretization/operator.cpp

```diff
@@ -272,6 +272,20 @@ Operator<dim, Number>::get_quad_index_gauss_lobatto() const
   return matrix_free_data->get_quad_index(get_quad_gauss_lobatto_name());
 }
 
+template<int dim, typename Number>
+double
+Operator<dim, Number>::compute_scaling_factor_mass(
+  double const scaling_factor_mass_from_acceleration,
+  double const scaling_factor_mass_from_velocity) const
```

Comment:
`scaling_factor_mass_from_velocity` gets multiplied by `weak_damping_coeficient`, so I am not sure if `scaling_factor_mass` is a good name. The product `param.weak_damping_coefficient * scaling_factor_mass_from_velocity` is in my opinion a scaling factor of the mass operator.

Reply:
I am not sure I get this comment -- you mean (i): the argument `scaling_factor_mass_from_velocity` has a misleading name since it is scaled with `weak_damping_coefficient` OR (ii) the local variable `scaling_factor_mass` should be renamed? My intention was to have one common scaling factor for the mass operator (hence `scaling_factor_mass` or maybe `total_...` or `final_...` or `summed_...`, which in my opinion is bit long and not really required).

At the same time, the arguments passed here originate from time integration only, they do not include any "physics" or any information regarding the operator. We get the scaling factors from the `time_integrator` via `time_integrator->get_scaling_factor_mass_from_acceleration()` and `time_integrator->get_scaling_factor_mass_from_velocity()`. From my current viewpoint, this split makes sense.

So I think the naming here actually makes sense, knowing the arguments are scaling factors from the time integration scheme only, and the total scaling factor of the mass operator _has_ to consider information on the actual operator, leading to the final `scaling_factor_mass`.